### ISSUE TITLE
Few ideas to vet

### ISSUE THREAD

**Issue by papasierragolf** [p2prider/p2prider#1]
1. Get oil and gas prices of the day from the websites of IOCL, HP, BP, Shell for metros and tier 1 cities - extrapolate them to price it amongst the tier 2 /3 towns local auto guys as well. 
This ensures that they get a fair price each day and can also discount or upscale their ride rates according to the daily prices.

2. Make an open google sheets, whenever you ride a cab, ask him for his number / email and as well 10 other friends of his. 
Or better yet, just share a link to a simple google form via their whatsapp and have them fill it up. 

3. No need to print brochure or QR codes initially. Possibly set up facebook and google ads targeted at drivers in local languages. Will require a bit of money to carry that out. Google adsense usually gives 2000rs to start with. We can be a bit stingy with the resources initially and select only few cities to start out with the ads. 
Post that we can integrate them with phone based qr codes for payment apps through our application or not - to ease the money transfer.

**Comment by p2prider** [comment]
Hey! Thanks for the suggestions. I've created two tracking tickets for the first point and second part of your last point.

> Make an open google sheets, whenever you ride a cab, ask him for his number / email and as well 10 other friends of his.
Or better yet, just share a link to a simple google form via their whatsapp and have them fill it up.

Yes I think this kind of outreach will be pretty good. I have started a wiki page on this which we can share with other members as a quick how-to: https://github.com/p2prider/p2prider/wiki/How-to-onboard-new-drivers-from-rides-from-other-platforms. Can you add more details to this wiki page and maybe create the google form as well and link it there?

Also, would be better to create separate issues in future for each idea/topic. Thanks a lot!

**Comment by papasierragolf** [comment]
How's the response so far?

**Comment by papasierragolf** [comment]
In Kolkata and I am facing the exactly same experience on uber / ola. 
Noone wants to do rental on uber here. 
Cabs want you to cancel and pay 50₹ less than uber if you are good with bargaining. The payment to their wallet or cash. 
So this problem of commissions is plaguing everyone.

**Comment by papasierragolf** [comment]
Also, we can utilise the padding spaces on app for banner ads - and keep the entire thing free

**Comment by ashwingattani** [comment]
> Also, we can utilise the padding spaces on app for banner ads - and keep the entire thing free

yeah, in the long run, we would need some revenue models to keep things going. Ads, sponsored incentives, donations are a few that can be worked upon.

**Comment by papasierragolf** [comment]
> > Also, we can utilise the padding spaces on app for banner ads - and keep the entire thing free
> 
> 
> 
> yeah, in the long run, we would need some revenue models to keep things going. Ads, sponsored incentives, donations are a few that can be worked upon.

I was speaking to few cab drivers. They are okay to pay a fixed monthly fee as well to avail the services. However, that should be irrespective of the number of rides they do. 
They are alleging that the current rate of commission for ride aggregator is 30%. I'm not sure if that is true however.

**Comment by ashwingattani** [comment]
> I was speaking to few cab drivers. They are okay to pay a fixed monthly fee as well to avail the services. However, that should be irrespective of the number of rides they do. 

This is a good idea, but we'll need to ensure they get decent rides per month to justify the subscription.

> They are alleging that the current rate of commission for ride aggregator is 30%. I'm not sure if that is true however.

Our self-enabled rate model would make this more transparent.

**Comment by papasierragolf** [comment]
> > I was speaking to few cab drivers. They are okay to pay a fixed monthly fee as well to avail the services. However, that should be irrespective of the number of rides they do. 
> 
> 
> 
> This is a good idea, but we'll need to ensure they get decent rides per month to justify the subscription.>
>
>
That subscription charges will come in later, but while signing up we tell them there will be subscription charges post demo period.

**Comment by papasierragolf** [comment]
As we get bigger, and have a budget, we can even add an OBD device to the car to monitor fuel efficiency etc and charge fares accordingly - this wil help to ensure the drivers also keep their cars at optimal performance levels, drive with a light foot etc to increase their efficiency and possibly we give them an analysis dashboard at the end of the month to review their vehicle performance. 
This is in order for them also to feel they are doing something good for the society
